ports/ocilib/portfile.cmake

```
@@ -0,0 +1,47 @@
+include(vcpkg_common_functions)
```

```suggestion
vcpkg_fail_port_install(ON_ARCH "arm" ON_TARGET "uwp")
````
Please also add this function.